app/presenters/check_answers/defendant_details_card.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module CheckAnswers
+  class DefendantDetailsCard < Base
+    attr_reader :defendant_details
+
+    def initialize(claim)
+      @defendant_details = claim.defendants
+      @group = 'about_defendant'
+      @section = 'defendant_summary'
+    end
+
+    def row_data
+      data = main_defendant_row
+      additional_defendants.each_with_index do |defendant, index|
+        data += additional_defendant_row(defendant, index)
+      end
+      data
+    end
```

Comment:
this can be simplified with flat_map (I think):
```
    def row_data
      main_defendant_row +
      additional_defendants.flat_map.with_index do |defendant, index|
        additional_defendant_row(defendant, index)
      end
    end
```